tutorials/classical_simulation/Simulating with laser noises.ipynb

```diff
@@ -0,0 +1,456 @@
+{
```

Comment:
<ul><li>Laser noises were introduced in the introductory tutorial <code>Simulating sequences with errors and noises</code>. They will be further detailed in this notebook. We recommend you to check the introductory tutorial before going through this one.</li><li><em>Laser noises model errors coming from the laser-atom interaction</em>. (It includes effects such as Doppler shift (the laser frequency no longer matches the transition due to the frequency shift induced by thermal motion), variation of the laser amplitude in space and between pulses. Hence ... :) -&gt; I don't think it's necessary, it's described below.</li></ul><p>Reminder of what I said in first notebook: </p><ul><li>In 2. Laser Noises: "The fluctuations of the properties of the lasers affect their effectiveness in addressing the desired transitions between atomic levels:"; I would also have "Therefore, the laser frequency <strong>they experience/addressed</strong> is shifted by Doppler <strong>shift/shift frequency/is Doppler-shifted</strong> due to thermal motion. This <strong>can be seen as</strong> a shift in the detuning frequency of the laser, and leads to a <strong>decrease </strong>in the Rydberg population." (we can count them) (does Rydberg population mean something or would "population of atoms in the Rydberg state" be more appropriate ?)</li></ul><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>